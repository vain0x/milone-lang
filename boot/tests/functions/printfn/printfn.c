int main();

int main() {
    printf("No formats\n");
    int call_ = 0;
    struct String hello_ = (struct String){.str = "Hello!", .len = 6};
    printf("%s\n", hello_.str);
    int call_1 = 0;
    printf("Welcome to %s. Entry price is %c%d!\n", "Milone Land", '$', 1000000007);
    int call_2 = 0;
    return 0;
}
