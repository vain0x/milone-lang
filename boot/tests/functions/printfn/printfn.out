No formats
Hello!
Welcome to Milone Land. Entry price is $1000000007!
$? = 0
